rhel/SPECS/pmm-server.spec

```diff
@@ -59,7 +58,6 @@ rm -rf %{pmm_repo}-%{pmm_commit}
 %doc README.md CHANGELOG.md
 %{_sysconfdir}/supervisord.d
 %{_sysconfdir}/alertmanager.yml
-%{_datadir}/percona-dashboards/import-dashboards.py*
```

Comment:
We need to bump up the changelog (like 2.24.0-1) and the release on line 10

Reply:
Yep, updated